virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc

```diff
@@ -0,0 +1,29 @@
+[id="virt-creating-data-volumes"]
+= Creating data volumes using profiles
+include::modules/virt-document-attributes.adoc[]
+:context: virt-creating-data-volumes
+
+toc::[]
+
+When you create a data volume, the Containerized Data Interface (CDI) creates a provisioned volume claim (PVC) and loads the PVC with your data. You can create a data volume as either a standalone resource or by using a virtual machine specification in a template.
+
+You create a data volume using the *pvc* or *storage* APIs,
+
+Not all storage provisioners are recognized by the CDI. However, you can configure automatic settings for a storage class if you customize the _storage profile_. The storage profile is a configuration file that you use to set storage allocation preferences. Using storage profiles enables you to create data volumes quickly while reducing coding, minimizing potential errors.
```

Comment:
The storage profile is a custom resource (CR) that you use to set storage allocation preferences.

Reply:
Perfect, Thanks.